klaviyo/SFCC_Klaviyo
# 202406 update viewed product

## Description
1. Removes option to brand Viewed Product and Add/Added to Cart events as SFCC in Klaviyo
2. Adds a new field to Viewed Product events `ProductID` which is compatible with Klaviyo's internal recommendations engine
3. updates the copy on the site configuration to specify that Viewed Product and Add/Added to Cart will never be branded.

<img width="1878" alt="image" src="https://github.com/klaviyo/SFCC_Klaviyo/assets/78871498/55818713-f284-4db9-b6f2-0a5140a6cf1a">




## Manual Testing Steps



1. Verified that when the option to brand events is selected, we still do not brand Viewed Product and Add / Added to Cart by testing all events. 
2. Verified that when the option to brand events is not selected, no events are branded. 
3. Verified that ProductID is included in the Viewed Product Payload
4. Ran unit tests, e2e tests to make sure all existing functionality is still intact. 

## Pre-Submission Checklist:

- [ ] You've updated the CHANGELOG following the steps [here](https://github.com/klaviyo/SFCC_Klaviyo#making-updates)

**NOTE:** Please use the [Changelogger](https://pypi.org/project/changelogged/) cli tool to manage versioned file upgrades.
